src/components/FormattedDateTime.js

```diff
@@ -2,18 +2,19 @@ import React from 'react';
 import { FormattedDate, FormattedTime } from 'react-intl';

```

Comment:
We should not have a separate component which is the same as the one exported by our shared component library.

Please remove this and change all imports of FormattedDateTime over to use stripes-erm-components.